.github/workflows/README.md

```diff
@@ -0,0 +1,89 @@
+# Info about how to use Github Actions
+
+## Adding data via a pull request from fork
+
+The main github action here is: NewDataPipeline.yml and it does two things:
+
+
+The new data pipeline relies on this workflow file to handle information from an incoming branch in a fork. It will first create a new branch and then open a pull request with the following setup:
+    
+1. First, create a new branch on this repository identical to the incoming branch from fork
+2. Then, open a pull request with the following setup:
+       
+    * The base of the new pull request will match the base of the fork's pull request.
+    * Example: If the fork is targeting the `main` branch with added files, this workflow creates a new branch in this repository and opens a pull request to the `main` branch of this repository.
+
+3. Then the AddData.yml workflow will be started with the info about relevant branches. This will go through the steps for adding experiment data first then the steps for adding simulation data. If no experiment data is present then those steps will be skipped. The scripts for adding experiment data and simulation data is in `Scripts/WorkflowScripts`. 
+The automated steps for sim data is described [here](https://nmrlipids.github.io/addingData.html).
+The automated steps for experiment data is described [here](https://nmrlipids.github.io/addingExpData.html)
+   * Experiment addition steps automated: 3,4,5
+   * Simulation addition steps automated: 4,5,6
+
+   * For the addition of simulation data the website automates even further by completing steps: 3,4,5,6,7
```

Comment:
❓ Her skriver vi at simulation data automatiserer steg 4,5,6 men rett under at den automatiserer 3,4,5,6,7. Eller leser jeg det feil?

Reply:
Den ene er for sim steps ved pull request, den andre er via nettsiden. Har endret det sånn at det er klarere for det var jo ikke det i utgangspunktet